### DISCUSSION TITLE
Why is this GPL

### DISCUSSION THREAD

**Discussion by davecramer** [sebasbaumh/postgis-java-ng#82] [General]
One of the strengths of PostgreSQL and by extension PostGIS is that it has a business friendly license.

By choosing to license this with GPL anyone using this project would have to deal with the consequences of this decision

**Comment by sebasbaumh** [comment]
This project is LGPL because the original base project [postgis-jdbc](https://github.com/postgis/postgis-java/tree/main/postgis-jdbc) is LGPL.

Actually I reworked a large part of the code to streamline it and bring it nearer to the actual PostGIS geometries in the database.
But I don't think I am allowed to put it under another license as long as I retain part of the original code.
Which is the reason I sticked to the LGPL.
Or do you know otherwise?

The company I work for also uses this library, which is allowed as the LGPL only affects the library code, but not the application using it.
After all I support your claim regarding a business friendly license though I do not think it is possible to change the license without having the authors of the original code change it.

**Comment by davecramer** [comment]
No, if you use any part of postgis-jdbc then you are stuck with LGPL. 

Thanks for your response
